src/TrackGenerator.cpp

```diff
@@ -1,5 +1,5 @@
 #include "TrackGenerator.h"
-
+#include<stdlib.h>
```

Comment:
I'm just curious, if it can be removed, why are you commenting on it rather
than removing it?
On Jun 17, 2015 12:08 PM, "Geoffrey Gunow" <notifications@github.com> wrote:

> In src/TrackGenerator.cpp
> <https://github.com/mit-crpg/OpenMOC/pull/157#discussion_r32662994>:
>
> > @@ -1,5 +1,5 @@
> >  #include "TrackGenerator.h"
> > -
> > +#include<stdlib.h>
>
> This can be removed
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-crpg/OpenMOC/pull/157/files#r32662994>.
>